src/quo2/components/drawers/bottom_actions/view.cljs

```diff
@@ -0,0 +1,39 @@
+(ns quo2.components.drawers.bottom-actions.view
+  (:require [quo2.components.buttons.button.view :as button]
+            [quo2.components.drawers.bottom-actions.style :as style]
+            [quo2.theme :as theme]
+            [quo2.components.markdown.text :as text]
+            [react-native.core :as rn]))
+
+(defn- bottom-actions-internal
+  "Options:
+
+  :actions - keyword (default nil) - :1-action/:2-actions
+  :description - string (default nil) - Description to display below the title
+  :button-one-label - string (default nil) - Label for the first button
+  :button-two-label - string (default nil) - Label for the second button
+  :theme - :light/:dark
+  :blur? - bool (default false)  - Boolean to set the blur color variant
+  :scroll - bool (default false) - Whether the iOS Home Indicator should be rendered"
+  [{:keys [actions description button-one-label button-two-label theme blur?
+           scroll]}]
+  [rn/view
+   [rn/view {:style style/buttons-container}
+    (when (= actions :2-actions)
+      [button/button {:size            40
+                      :container-style style/button-container-2-actions
+                      :type            :grey
+                      :background      :blur} button-one-label])
```

Comment:
it seems `background` should only be `:blur` when `blur?` is true

Reply:
i.e `:background (when scroll? :blur)`

Reply:
Hey correct me if I'm wrong — seems like the background is not dependent on scroll prop.
<img width="1691" alt="image" src="https://github.com/status-im/status-mobile/assets/15213331/769b1bd8-ee24-497f-9b13-cbb62e002939">